arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -683,6 +670,32 @@
         "type"
       ]
     },
+    "ExampleRequest": {
+      "type": "object",
+      "description": "Sample input data for the service's input(s).",
+      "properties": {
+        "inputs": {
+          "type": "object",
+          "description": "Sample input data for the web service's input(s) given as an input name to sample input values matrix map.",
+          "additionalProperties": {
```

Comment:
In Swagger world, additionalProperties is used to model [maps/dictionaries](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#model-with-mapdictionary-properties). Using additionalProperties to model as an array is so confusing for the end user